packages/api-fetch/package.json

```diff
@@ -24,7 +24,8 @@
 		"@babel/runtime": "^7.0.0",
 		"@wordpress/hooks": "file:../hooks",
 		"@wordpress/i18n": "file:../i18n",
-		"@wordpress/url": "file:../url"
+		"@wordpress/url": "file:../url",
+		"lodash": "^4.17.10"
```

Comment:
I'm not strictly opposed to it since it's a library which in a wp-admin context will most certainly be loaded anyways, it's worth pointing out that for the npm distributable, this has a sizeable impact in bundle size, for what effectively amounts to a usage of what can otherwise be expressed as `() => {}`.

The "called with" test, if an argument for having something with referential equality, is not one I think we need to guarantee anyways. The guarantee is simply that the middleware is passed a function (`expect.any( Function )`).

Reply:
👍 I think we can avoid avoiding pulling all of lodash as a requirement here, especially since we're only using noop. @mmtr generally we need to be a bit more careful on the trade-offs costs of pulling packages into libraries vs applications.

Reply:
In theory this becomes a non-issue if we revisit something like #9374